packages/mattermost-redux/src/selectors/entities/roles.ts

```diff
@@ -50,6 +50,43 @@ export const getMyGroupRoles: (state: GlobalState) => Record<string, Set<string>
     },
 );
 
+export const getGroupListPermissions: (state: GlobalState) => Record<string, GroupPermissions> = createSelector(
```

Comment:
It would be nice to add some documentation to this, for example:

```suggestion
/** Returns a map of permissions—keyed by group id—for groups in state that are not deleted, mentionable, and that the current user is a member of.*/
export const getGroupListPermissions: (state: GlobalState) => Record<string, GroupPermissions> = createSelector(
```

Reply:
Added a comment